packages/hammer/src/setup.ts

```diff
@@ -1,26 +1,30 @@
 import { Managers, Transactions } from "@arkecosystem/crypto";
+import { ProtokolConnection } from "@protokol/client";
 import { Transactions as NFTTransactions } from "@protokol/nft-base-crypto";
 import { Transactions as NFTExchangeTransactions } from "@protokol/nft-exchange-crypto";
 
 import { configurations } from "./configurations";
-import exceptions from "./data/networks/exceptions.json";
-import milestones from "./data/networks/milestones.json";
-import configs from "./data/networks/network.json";
 
 export const setupScript = async () => {
-	Managers.configManager.setFromPreset(configurations.network as "testnet" | "devnet" | "mainnet");
-	Managers.configManager.setHeight(configurations.networkHeight);
+	const client = new ProtokolConnection(configurations.clientHost);
+	const configs = await client.api("node").crypto();
 
 	Managers.configManager.setConfig({
-		network: configs,
-		milestones: milestones,
+		network: configs.body.data.network,
+		milestones: configs.body.data.milestones,
 		genesisBlock: Managers.configManager.getPreset("devnet").genesisBlock,
-		exceptions: exceptions,
+		exceptions: configs.body.data.exceptions,
 	});
+	Managers.configManager.setHeight(2);
```

Comment:
Use correct height from node.
```suggestion
        const { body: { data: { block : { height } } } }= await client.get("blockchain");
	Managers.configManager.setHeight(height );
```